src/editor/Model/EditorGroup.re

```diff
@@ -6,7 +6,10 @@
 
 open Oni_Core;
 
+let _latestId = ref(0);
```

Comment:
side note: we have `Revery.UniqueId` so you could use `let editorGroupId = Revery.UniqueId.make({})` then `editorGroupId.getUniqueId` 🤷‍♂️

Reply:
Good point @Akin909 , I'll switch to that 👍